README.md

```diff
@@ -268,6 +269,9 @@ The following methods can be called on the jstree:
 
 * `grid_hide_column(column)`: Hide column number `column`. Column numbers begin with `0`. If the column already is hidden, has no effect.
 * `grid_show_column(column)`: Show column number `column`. Column numbers begin with `0`. If the column already is shown, has no effect.
+* `search(str)`: the [jstree search](https://www.jstree.com/api/#/?q=search) has been extended to search the entire grid. to search a specific column use `searchColumn`.
```

Comment:
I've actually been thinking about this line. I feel like we should mention that we've extended it but with the current implementation it actually only searches the entire grid when the user hasn't overridden the callback function. is this something we want to change so that a changed search callback by the user also searches the entire grid? Or should I just mention this in the readme